common/src/API.php

```diff
@@ -63,6 +63,26 @@ private function buildURL($address, $parameters) {
         }
         return $address . '&' . $query_string;
     }
+
+    public function hasConsent(array $data): bool {
```

Comment:
`$data` can be `$order_number`
```suggestion
    public function hasConsent(array $order_number): bool {
```